polaris-core/src/main/java/org/apache/polaris/core/PolarisConfigurationStore.java

```diff
@@ -69,6 +69,16 @@ public interface PolarisConfigurationStore {
 
     if (config.defaultValue instanceof Boolean) {
       return config.cast(Boolean.valueOf(String.valueOf(value)));
+    } else if (config.defaultValue instanceof Short) {
```

Comment:
I think we can leave off Short here. When would you make a config a short over an int?

Reply:
Updated